pyvo/dal/adhoc.py

```diff
@@ -597,7 +619,7 @@ def clone_byid(self, id):
         for x in copy_tb.resources:
             if x.ID and x.ID not in referenced_serviced:
                 copy_tb.resources.remove(x)
-        return DatalinkResults(copy_tb)
+        return DatalinkResults(copy_tb, original_row=original_row)
```

Comment:
shouldn't original row be a copy as well, from whatever is already set in the class, in `self.original_row`?